src/MUIDataTable.js

```diff
@@ -16,8 +16,8 @@ import TableHead from './components/TableHead';
 import TableResize from './components/TableResize';
 import TableToolbar from './components/TableToolbar';
 import TableToolbarSelect from './components/TableToolbarSelect';
-import getTextLabels from './textLabels';
-import { buildMap, getCollatorComparator, sortCompare, getPageValue } from './utils';
+import textLabels from './textLabels';
```

Comment:
Is there a reason this line was changed? We need the `getTextLabels` import from what I can tell.

Reply:
I actually dont remember, I'll go back and revert it.